spot-integrations-service/README.md

```diff
@@ -0,0 +1,12 @@
+# Spot Integrations Service
+
+A backend service for signing Zoom meeting numbers with an api secret and api key.
+
+## Getting Started
+1. `npm install`
+1. `npm start`
+
+## Building
+1. `chmod +x ./scripts/build.sh`
```

Comment:
is this step really required? why not include executable in git?

Reply:
I guess then I already pushed it as executable. I don't know how file permission persistence works. I'll update the readme.